Notify/Notify.Functions/Notify.Functions/NotifyFunctions/Users/GetUserByUserName.cs

```diff
@@ -0,0 +1,64 @@
+using System;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Azure.WebJobs;
+using Microsoft.Azure.WebJobs.Extensions.Http;
+using Microsoft.AspNetCore.Http;
+using Microsoft.Extensions.Logging;
+using MongoDB.Bson;
+using MongoDB.Driver;
+using Notify.Functions.Core;
+using Notify.Functions.HTTPClients;
+
+namespace Notify.Functions.NotifyFunctions.Users
+{
+    public static class GetUserByUserName
+    {
+        [FunctionName("GetUserByUserName")]
+        [AllowAnonymous]
+        public static async Task<IActionResult> RunAsync(
+            [HttpTrigger(AuthorizationLevel.Anonymous, "get", Route = "user/{userName}")]
+            HttpRequest req, string userName, ILogger log)
+        {
+            IMongoCollection<BsonDocument> collection;
+            FilterDefinition<BsonDocument> userFilter;
+            BsonDocument user;
+            ObjectResult result;
+            ProjectionDefinition<BsonDocument> projection;
+
+            try
+            {
+                log.LogInformation($"Received request to get user profile for username: {userName}");
+
+                collection = AzureDatabaseClient.Instance.GetCollection<BsonDocument>(
```

Comment:
I will take a look

Reply:
I wasn't rebased on master and you merged.
We need a big refactor there because every azure function uses the old way, so I will do that in a different PR.